addons/mail/static/src/models/messaging.js

```diff
@@ -93,6 +93,9 @@ addRecordMethods('Messaging', {
         const { message, ...options } = params;
         return this.env.services.notification.add(message, options);
     },
+    onClickEmoji(ev) {
+        this.messaging.messagingBus.trigger('add_emoji', { emoji: ev.currentTarget.dataset.codepoints });
+    },
```

Comment:
Should maybe guard `this.messaging`, any handler might be called on non-existing record